tests/scripts/thread-cert/Cert_7_1_07_BorderRouterAsLeader.py

```diff
@@ -0,0 +1,540 @@
+        #                      - 6LoWPAN ID sub-TLV
+        #                          Stable Flag set
+        _dr_pkt = pkts.filter_wpan_src64(LEADER).\
+            filter_LLANMA().\
+            filter_mle_cmd(MLE_DATA_RESPONSE).\
+            filter(lambda p: {
+                              NETWORK_DATA_TLV,
+                              SOURCE_ADDRESS_TLV,
+                              LEADER_DATA_TLV
+                             } <= set(p.mle.tlv.type) and\
+                   [Ipv6Addr(PREFIX_1[:-3])] ==
+                   p.thread_nwd.tlv.prefix
+                   ).\
+            must_next()
+        _dr_pkt1 = pkts.filter_wpan_src64(LEADER).\
+            filter_LLANMA().\
+            filter_mle_cmd(MLE_DATA_RESPONSE).\
+            filter(lambda p: {
+                              NETWORK_DATA_TLV,
+                              SOURCE_ADDRESS_TLV,
+                              LEADER_DATA_TLV
+                             } <= set(p.mle.tlv.type) and\
+                             {
+                              NWD_BORDER_ROUTER_TLV,
+                              NWD_BORDER_ROUTER_TLV,
+                              NWD_6LOWPAN_ID_TLV
+                             } <= set(p.thread_nwd.tlv.type) and\
+                   set(p.thread_nwd.tlv.stable) == {0, 1, 1, 1, 0} and\
+                   p.mle.tlv.leader_data.data_version ==
+                   (_dr_pkt.mle.tlv.leader_data.data_version + 1) % 256 and\
+                   [Ipv6Addr(PREFIX_1[:-3])] ==
+                   p.thread_nwd.tlv.prefix
+                   ).\
+            must_next()
+
+        # Step 6: Leader MUST send a MLE Child Update Request or MLE Data
+        #         Response to SED, including the following TLVs:
+        #             - Network Data TLV
+        #                 At least one Prefix TLV (Prefix 1) including:
+        #                     - Stable Flag set
+        #                     - Border Router sub-TLV(corresponding to Router_1)
+        #                         - P_border_router_16<0xFFFE>
+        #                         - Stable Flag set
+        #                     - 6LoWPAN ID sub-TLV
+        #                         - Stable Flag set
+        #             - Source Address TLV
+        #             - Leader Data TLV
+        #             - Active Timestamp TLV
+        pkts.filter_wpan_src64(LEADER).\
+            filter_wpan_dst64(SED).\
+            filter_mle_cmd2(MLE_CHILD_UPDATE_REQUEST, MLE_DATA_RESPONSE).\
+            filter(lambda p: {
+                              NETWORK_DATA_TLV,
+                              SOURCE_ADDRESS_TLV,
+                              LEADER_DATA_TLV,
+                              ACTIVE_TIMESTAMP_TLV
+                             } == set(p.mle.tlv.type) and\
+                   [Ipv6Addr(PREFIX_1[:-3])] ==
+                   p.thread_nwd.tlv.prefix and\
+                   p.thread_nwd.tlv.stable == [1, 1, 1] and\
```

Comment:
thread_nwd.tlv.stable is a list of stable flag in NetworkData, in this filter, since all stable flag are set, these can be checked like == [1, 1, 1] ,  otherwise, stable list needs to refer to the Prefix's and its BorderRouter 's order in the packet

Reply:
add 6lowpan ID compress flag checks to help distinguish

Reply:
Nice to have 6LoWPAN ID compress flag check, however my concern is the way to check stable flag here seems not flexible enough and may have issues in some situations: 
Like if we want to run the test scripts on OTBR1.2 which introduces BBR Dataset which may contribute the Service TLV and Server TLV into the Network Data, that is, there would be two additional stable flags accordingly.

The best way to make the script more robust is to simply check the content required by the Test Plan without assuming there are no other server info, that also explains the reason that Test Plan introduces `At least`.
Ownership/affiliation should be paid attention when checking. That is, for this step, we should check Prefix_1 appears in the Network data with stable flag set, and its sub TLVs (BorderRouter TLV and 6LoWPAN TLV appears, both with their stable flag set. and P_border_router_16 of BorderRouter TLV has value 0xFFFE.
Similarly for the other steps in this test.

Thoughts?